### ISSUE TITLE
Appimage

### ISSUE THREAD

**Issue by rapha8l** [minbrowser/min#567]
Hi, could it be possible to have an AppImage https://en.wikipedia.org/wiki/AppImage
This would allow all Linux users to use Min without problems.
Thanks

**Comment by code-hunger** [comment]
I'm a Linux user and have no problems with Min. What problems do you have that AppImage will solve?

**Comment by rapha8l** [comment]
Running Min without having to build if you are not Debian or Os X user.
AppImage run on all glibc Linux.
It's not exactly a problem, but more an improvement.

**Comment by code-hunger** [comment]
So you suggest this thing just to avoid running `grunt` before starting min? I don't see running a single command as such a problem that needs introducing a packaging system, though for some lazy users it may be easier to download a file and click on it, instead of download a repo and `grunt` it.

**Comment by PalmerAL** [comment]
Building from source can be kind of a pain though, so if there are ways we can make it easier to avoid that it might be worth looking into (and I wouldn't really call it "lazy"). I don't know much about appimage, I'm traveling for the next few weeks but I'll try to look into it after that.

**Comment by rapha8l** [comment]
Thanks

**Comment by verbalius** [comment]
This is even easier because the project is on electron and from the AppImage documentation there is a electronbuilder which can automatically build AppImages from source. Here is a link to it https://docs.appimage.org/packaging-guide/overview.html#using-electron-builder

**Comment by tudorels** [comment]
It would be nice for many users to have appimage of min, not all of them are able to build this ... So, can you consider releasing an appimage? Thanks!

**Comment by maverick74** [comment]
I would totally vote for this as well!

AppImage is a great way to test and use Software.

I have a few appimages of software i don't really want to have installed!
(beside... i can just put it on a USB Drive and "go"...)

please consider it :)

**Comment by aadilayub** [comment]
Yes, this would be great for users on non-Debian or non Fedora-based distributions. It will also help users of immutable distributions

**Comment by psython123** [comment]
Voting for this option.
